src/main/kotlin/anttracker/PageOf.kt

```diff
@@ -0,0 +1,119 @@
+package anttracker
+import org.jetbrains.exposed.sql.SizedIterable
+import org.jetbrains.exposed.sql.transactions.transaction
+import kotlin.math.ceil
+
+// A PageOf objects contains a max-20 element list of T.
+// Provides top-level functionality for:
+//  - loading DB records into memory (one page at a time)
+//  - accessing records
+//  - displaying records as a page
+//  - going to the next page and loading its records
+abstract class PageOf<IntEntity> {
```

Comment:
I think that we want any `T` that extends the `IntEntity` class. Also, perhaps using `Entity` is enough.

Reply:
changed to T: IntEntity